source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -104,17 +104,10 @@ Extractor::extractValue(Http::StreamFilterCallbacks &callbacks,
   return "";
 }
 
-TransformerInstance::TransformerInstance(
-    const Http::RequestOrResponseHeaderMap &header_map,
-    const Http::RequestHeaderMap *request_headers, GetBodyFunc &body,
-    const std::unordered_map<std::string, absl::string_view> &extractions,
-    const json &context,
-    const std::unordered_map<std::string, std::string> &environ,
-    const envoy::config::core::v3::Metadata *cluster_metadata,
-    Envoy::Random::RandomGenerator &rng)
-    : header_map_(header_map), request_headers_(request_headers), body_(body),
-      extractions_(extractions), context_(context), environ_(environ),
-      cluster_metadata_(cluster_metadata), rng_(rng) {
+// A TransformerInstance is constructed by the InjaTransformer::transform method
```

Comment:
Is this still true? It seems like it's created by the injaTransformer constructor now

Reply:
Good catch, thanks!